recipes/fmt/all/conandata.yml

```diff
@@ -50,10 +50,7 @@ sources:
 patches:
   "5.3.0":
     - patch_file: "patches/fix-install-5.3.0.patch"
-      base_path: "source_subfolder"
```

Comment:
Does it mean than patches and source code are mixed together in the same folder now? Is it a good idea?

Reply:
Good comment.
There is a good argument to store the project sources in some subfolder.
It also allows to have multiple source folders if some project consists of multiple root source folders.

Reply:
No, the patches go to the source root folder (as previously) but the location of the sources is managed by the layout method and the `folders.source`